PigLatin/PigLatin.cs

```diff
@@ -7,15 +7,125 @@ class Program
+            Console.WriteLine(lowercaseNewWord);
+            Console.WriteLine("");
+
+            Console.WriteLine("Let's try a whole sentence!");
+            string userInputSentence = Console.ReadLine();
+            string translatedSentence = moreWords(userInputSentence);
+            Console.WriteLine(translatedSentence);
+            Console.Write("oops let me fix that punctuation");
+            for (int i = 0; i < 5; i++)
+            {
+                System.Threading.Thread.Sleep(1000);
+                Console.Write(".");
+            }
+            Console.WriteLine("");
+            string fixTranslation = fixPunctuation(translatedSentence);
+            Console.WriteLine(fixTranslation);
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
         }
-        
+
         public static string TranslateWord(string word)
+        {
+            string firstLetter = word.Substring(0, 1);
+            string restWord = word.Substring(1);
+            string translatedWord = (restWord + firstLetter + "ay");
+            return translatedWord;
+        }
+
+        public static string TranslateWordVowel(string word)
         {
             // your code goes here
-            return word;
+            int firstVowelIndex = word.IndexOfAny(new char[] { 'a', 'e', 'i', 'o', 'u', 'y' });
+            string firstPart = word.Substring(0, firstVowelIndex);
+            string secondPart = word.Substring(firstVowelIndex);
+            string translatedWord = (secondPart + firstPart + "ay");
+
+            return translatedWord;
+        }
+
+        public static string startsWithVowel(string word)
+        {
+            // your code goes here
+            int firstVowelIndex = word.IndexOfAny(new char[] { 'a', 'e', 'i', 'o', 'u', 'y' });
+            string last = word.Substring(0, 1);
+            string first = word.Substring(1);
+            if (firstVowelIndex == 0)
+            {
+
+                string translatedWord = (last + first + "yay");
+
+                return translatedWord;
+            }
+            else
+            {
+                string notAVowel = (first + last + "ay (try a word that starts with a vowel!)");
+                return notAVowel;
+            }
+
+        }
+        public static string makeLowerCase(string word)
+        {
+            string convertLower = word.ToLower();
+            string firstLetter = convertLower.Substring(0, 1);
+            string restWord = convertLower.Substring(1);
+            string translatedWord = (restWord + firstLetter + "ay (try a word that has differant cased letters!)");
+            return translatedWord;
+        }
+        public static string moreWords(string words)
+        {
+            string[] allWords = words.Split(' ');
+            string[] translatedWords = new string[allWords.Length];
+            for (int i = 0; i < allWords.Length; i++)
```

Comment:
Why not use some of your functions from above here?

Reply:
All the functions were written before I realized a lot of things (mostly things about reusable code) then I  haphazardly wrote them all together. It came due before I could clean it up. If i was to write it again it would be A lot shorter ha.